### ISSUE TITLE
Issue when Loading the Options file 

### ISSUE THREAD

**Issue by amkhullar** [codingo/NoSQLMap#87]
## What's the problem (or question)?
Get an error while opening the options file from nosqlmap

## How can we reproduce the issue?

C:\NoSQLMap-master>python nosqlmap.py
'clear' is not recognized as an internal or external command,
operable program or batch file.
 _  _     ___  ___  _    __  __
| \| |___/ __|/ _ \| |  |  \/  |__ _ _ __
| .` / _ \__ \ (_) | |__| |\/| / _` | '_ \
|_|\_\___/___/\__\_\____|_|  |_\__,_| .__/
 v0.7 codingo@protonmail.com        |_|

1-Set options
2-NoSQL DB Access Attacks
3-NoSQL Web App attacks
4-Scan for Anonymous MongoDB Access
5-Change Platform (Current: MongoDB)
x-Exit
Select an option: 1

Options
1-Set target host/IP (Current: Not Set)
2-Set web app port (Current: 80)
3-Set App Path (Current: Not Set)
4-Toggle HTTPS (Current: OFF)
5-Set MongoDB Port (Current : 27017)
6-Set HTTP Request Method (GET/POST) (Current: GET)
7-Set my local MongoDB/Shell IP (Current: Not Set)
8-Set shell listener port (Current: Not Set)
9-Toggle Verbose Mode: (Current: OFF)
0-Load options file
a-Load options from saved Burp request
b-Save options file
h-Set headers
x-Back to main menu

Select an option: 0
Enter file name to load: c:\burp

Traceback (most recent call last):
  File "nosqlmap.py", line 544, in <module>
    main(args)
  File "nosqlmap.py", line 47, in main
    mainMenu()
  File "nosqlmap.py", line 81, in mainMenu
    options()
  File "nosqlmap.py", line 419, in options
    optList = csvOpt[0].split(",")
NameError: global name 'csvOpt' is not defined

## What are the running context details?

* Installation method (e.g. `pip`, `apt-get`, `git clone` or `zip`/`tar.gz`): zip
* Client OS (e.g. `Microsoft Windows 10`) Windows 10
* Program version (`python sqlmap.py --version` or `sqlmap --version` depending on installation): 0.7
* Target DBMS (e.g. `Mongo`): Mongo
* Detected WAF/IDS/IPS protection (e.g. `ModSecurity` or `unknown`):
* Results of manual target assessment
* Relevant console output (if any):
* Exception traceback (if any):

**Comment by codingo** [comment]
If you update the following lines:

```
    while mmSelect:
        os.system('clear')
```

To read:

```
    while mmSelect:
        os.system('cls')
```

This should correct this. If you can let me know if it does and I"ll replace this with a more correct way of wiping the buffer, if there's a chain of regular issues after this I'm likely to just cast the project as having Linux support only and opt to close this. Could you make this change and let me know what happens?

**Comment by amkhullar** [comment]
which file is this ?

**Comment by amkhullar** [comment]
That issue is fixed with 'cls' but the other issue was not fixed. 

`Traceback (most recent call last):
File "nosqlmap.py", line 544, in 
main(args)
File "nosqlmap.py", line 47, in main
mainMenu()
File "nosqlmap.py", line 81, in mainMenu
options()
File "nosqlmap.py", line 419, in options
optList = csvOpt[0].split(",")
NameError: global name 'csvOpt' is not defined`

**Comment by codingo** [comment]
Closed with #95
